octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -707,7 +743,18 @@ async def order(self, order: cryptofeed_types.OrderInfo, receipt_timestamp: floa
         :param order the order object defined in cryptofeed.types.?
         :param receipt_timestamp: received timestamp
         """
-        await self.push_to_channel(trading_constants.ORDERS_CHANNEL, orders=[self.exchange.parse_order(order)])
+        await self.push_to_channel(trading_constants.ORDERS_CHANNEL, orders=[{
+            trading_enums.ExchangeConstantsOrderColumns.TYPE.value: self._parse_order_type(order.type),
```

Comment:
should we parse these data (order type etc) ? I think what we want here is to convert it into a format similar to the order updater and it will be parsed later on in Order

Reply:
I don't understand, it's exactly what we are doing here : converting cryptofeed format to a parsable format for order updater or order.

Reply:
`_parse_order_type` returns a `trading_enums.TradeOrderType`, order updaters return a `str`, parsing of this `str` is made later in `Order`. Unless I missed something, I think we will have a different format in rest and in ws if we keep this (for the content of the order dict we provide to the order channel)

Reply:
`_parse_order_type` is returning a `str` in this merge request.

Reply:
Oh right, my bad !